pkg/console/operator/operator.go

```diff
@@ -239,15 +239,9 @@ func (c *consoleOperator) handleSync(configs configSet) error {
 	c.ConditionsDefault(updatedStatus)
 	err := c.sync_v400(updatedStatus, configs)
 
-	c.HandleDegraded(updatedStatus, "SyncError", func() error {
```

Comment:
`SyncErrors` are not real errors, from the beginning, we have used this as a way to just kick the sync loop.  They really shouldn't be reported at all.

The problem here now is that `CustomLogo` is the ONLY degraded status we report.  Everything else is `Progressing`. 

We probably need to reasses what we assign to `Progressing` and decide if that is actually `Degraded`.

Reply:
We used to say `if !IsSyncError && !IsCustomLogoError` then we are `Degraded` with the error as a `message`.  The problem is that we report too many `GenericSyncErrors` and they don't mean anything (since the message doesn't make it into Telemetry from Monitoring).  Not sure at the moment how to fix this.  Its not meaningful to track an error if we don't know what the error is.  BUT we shouldn't ignore them and let it look like everything is fine.